routingtable/adjRIBIn/adj_rib_in.go

```diff
@@ -14,16 +15,18 @@ import (
 // AdjRIBIn represents an Adjacency RIB In as described in RFC4271
 type AdjRIBIn struct {
 	routingtable.ClientManager
-	rt           *routingtable.RoutingTable
-	mu           sync.RWMutex
-	exportFilter *filter.Filter
+	rt               *routingtable.RoutingTable
```

Comment:
Nit: the naming of these is not consistent: `RoutingTable` gets shortened to `rt`, but then there's a long `exportFilter` and `constibutingASNs`...